client/layout/error.jsx

```diff
@@ -48,5 +50,12 @@ export function retry( chunkName ) {
 export function show( context, chunkName ) {
 	log( 'Chunk %s could not be loaded', chunkName );
 	analytics.mc.bumpStat( 'calypso_chunk_error', chunkName );
+	context.store.dispatch( {
+		type: SECTION_SET,
+		section: false,
+		hasSidebar: false,
+	} );
 	context.primary = <LoadingErrorMessage />;
+	makeLayout( context, noop );
+	clientRender( context );
```

Comment:
I guess we call `clientRender` at this place because _error_ is not a section by itself and doesn't have a corresponding `controller` which would call `render()` eventually?